src/components/Header.js

```diff
@@ -24,7 +24,7 @@ export default function Header({ logo, showClose, title, ...props }) {
   const { update } = React.useContext(Context)
 
   return (
-    <HeaderStyle backgroundColor={props.color || 'none'}>
+    <HeaderStyle id={'header'} backgroundColor={props.color || 'none'}>
```

Comment:
if this is for cypress testing the good idea is to prefix with soething like `cy-` https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Done